charts/terraform-pod/Chart.yaml

```diff
@@ -6,7 +6,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.0.8
+version: 0.0.9
```

Comment:
in your feature/branches you can only use -dev.x suffix
https://github.com/saritasa-nest/saritasa-devops-helm-charts/blob/88fe216c92dc22a559f660f8a2c50dfab72194fb/charts/tekton-pipelines/Chart.yaml#L9

otherwise you break compilation cycle when you merge to `main` branch

so 
1. delete this build
2. delete tags/release
3. rename to 0.0.9-dev.1 and test it.
4. reopen PR
5. when approved - merge it into main (this will fail the github actions build)
6. in the main branch - change the version to `0.0.9` and push to main (this should make the proper build/release without any errors if you made step 2 above properly)

Reply:
step 1-4 are complited